.github/workflows/pr-md-link-check.yaml

```diff
@@ -20,4 +20,4 @@ jobs:
         use-quiet-mode: 'yes'
         config-file: .markdownlinkcheck.json
         check-modified-files-only: 'yes'
-        base-branch: main
+        base-branch: release-1.10
```

Comment:
If I'm not wrong, this change should apply to the release branch only, not to main

Reply:
Yes, that was quite the mistake. I'll re-do this.